cmd/grr/main.go

```diff
@@ -33,12 +33,30 @@ func main() {
 	if err != nil {
 		log.Fatalln(err)
 	}
-	registry := grizzly.NewRegistry(
-		[]grizzly.Provider{
-			grafana.NewProvider(&context.Grafana),
-			mimir.NewProvider(&context.Mimir),
-			syntheticmonitoring.NewProvider(&context.SyntheticMonitoring),
-		})
+	providers := []grizzly.Provider{}
+
+	grafanaProvider, err := grafana.NewProvider(&context.Grafana)
+	if err != nil {
+		log.Warnf("Grafana provider is not configured: %v", err)
```

Comment:
Do we want to warn, or silently ignore? I always feel uncomfortable Grizzly listing the ignored providers. Perhaps we could list the *configured* providers rather than the ignored ones?

Reply:
I often feel uncomfortable seeing the skipped handlers. I told it I don't want to use them, so strictly speaking it isn't "skipping" them. Same deal here?